src/main/kotlin/no/nav/familie/ef/søknad/mapper/SøkerinfoMapper.kt

```diff
@@ -1,13 +1,41 @@
 package no.nav.familie.ef.søknad.mapper
 
 import no.nav.familie.ef.søknad.api.dto.Søkerinfo
+import no.nav.familie.ef.søknad.api.dto.tps.Adresse
+import no.nav.familie.ef.søknad.api.dto.tps.Barn
+import no.nav.familie.ef.søknad.api.dto.tps.Person
+import no.nav.familie.ef.søknad.integration.dto.AdresseinfoDto
 import no.nav.familie.ef.søknad.integration.dto.PersoninfoDto
 import no.nav.familie.ef.søknad.integration.dto.RelasjonDto
 
 object SøkerinfoMapper {
 
     fun mapTilSøkerinfo(personinfoDto: PersoninfoDto, aktuelleBarn: List<RelasjonDto>): Søkerinfo {
-        return Søkerinfo(PersonMapper.mapTilPerson(personinfoDto),
```

Comment:
Flytter funksjoner fra PersonMapper inn hit for å samle mapping av person-/barndata fra tps-objekt til søkerinfo-objekt som returneres til frontend. 
Også for å tydeliggjøre mapping av persondata inn (søkerinfomapper ut til frontend) og ut (personaliamapper inn til interne systemer)